app/src/components/features/rules/RuleBuilder/index.js

```diff
@@ -343,10 +351,41 @@ const RuleBuilder = (props) => {
         />
       ) : null}
 
-      <Body
-        mode={MODE}
-        currentlySelectedRuleConfig={currentlySelectedRuleConfig}
-      />
+      <Row className="w-full relative">
+        <Col span={showDocs ? 17 : 24}>
+          <Body
+            mode={MODE}
+            showDocs={showDocs}
```

Comment:
Yes, it does exactly that but we also need to shrink the title component and give more span for rule pairs.

Reply:
Rule title and Help button can be in a single row with flex space-between. If Help is clicked, it will notify the parent, so that it can shift the editor to the left and open the docs panel on the right.